include/tss2/tss2_tcti_i2c_ftdi.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 Infineon Technologies AG
```

Comment:
Use SPDX Licence header. Note that the license you provided is not BSD-2.

/* SPDX-License-Identifier: BSD-2-Clause */
/*
 * Copyright (c) 2023 Infineon Technologies AG
 * All rights reserved.
 */

Reply:
fixed all license header.